packages/blade-mcp/knowledgebase/PasswordInput.md

```diff
@@ -0,0 +1,191 @@
+## Component Name
```

Comment:
I've already added PasswordInput in #2653

Reply:
<img width="480" alt="image" src="https://github.com/user-attachments/assets/30160b84-b1d0-4e17-8c1f-9bd0f4992f26" />

There's no PasswordInput 🙈